assets/js/2pulumi.js

```diff
@@ -217,7 +227,7 @@ function convertCode(language, inputKind) {
                 $("#pulumi-warnings").show();
             }
         })
-        .fail(function(err) {
+        .fail(function (err) {
```

Comment:
Looks like vscode applied some formatting. If this is inconsistent I am happy to revert the formatting changes.

Reply:
It's not consistent, but it's okay; there's more good here than not. Out of curiousity, what formatter are you using?